plugins/virt.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: virt
+spec:
+  version: "v0.17.2"
+  platforms:
+    - selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+      uri: "https://github.com/dhiller/kubectl-virt-plugin/releases/download/v0.17.2/virtctl-v0.17.2-linux-amd64.tar.gz"
+      sha256: "bbf3b4f15b64ffe4c9196b53d045a92bc8113d24cfde471dded3c8c505aef663"
+      files:
+        - from: "/virtctl/virtctl-v0.17.2-linux-amd64"
+          to: "."
+      bin: "./virtctl-v0.17.2-linux-amd64"
+    - selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+      uri: "https://github.com/dhiller/kubectl-virt-plugin/releases/download/v0.17.2/virtctl-v0.17.2-windows-amd64.exe.tar.gz"
+      sha256: "828876ae8abc50c7965f7b9eaa1ff3f6d3d9b0dad25c79e6a7d49dd6a1543087"
+      files:
+        - from: "/virtctl/virtctl-v0.17.2-windows-amd64.exe"
+          to: "."
+      bin: "./virtctl-v0.17.2-windows-amd64.exe"
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+      uri: "https://github.com/dhiller/kubectl-virt-plugin/releases/download/v0.17.2/virtctl-v0.17.2-darwin-amd64.tar.gz"
+      sha256: "ffdc599d4581a98e230ffab30cde39a20db3a695d300d726f72bbdebd9a57361"
+      files:
+        - from: "/virtctl/virtctl-v0.17.2-darwin-amd64"
+          to: "."
+      bin: "./virtctl-v0.17.2-darwin-amd64"
+  shortDescription: controls virtual machine related operations.
+  homepage: https://kubevirt.io
+  caveats: |
+    Usage:
+      kubectl virt
+
+    Documentation:
+      https://kubevirt.io/user-guide/docs/latest/administration/intro.html#client-side-virtctl-deployment
+  description: |
+    virt plugin controls virtual machine related operations on your kubernetes cluster.
```

Comment:
Here, you should also mention that this plugin is a wrapper to make `virtctl` available as `kubectl virt`.

Besides, the description is what sells your plugin. Interested users will see the description when running `krew info virt`. Therefore you should provide all meaningful info about it, so that users understand why they would want it.

Reply:
Interesting. I tried to check what that looks like by typing 

    $ kubectl krew install --manifest virt.yaml \
        --archive=virtctl-v0.17.2-linux-amd64.tar.gz
    $ kubectl krew info virt

And got this result: 

    F0607 09:48:58.882109   11512 root.go:52] plugin "virt" not found

Does that relate to the plugin being installed only locally and not yet present in the krew-index or is this a bug?

Reply:
> And got this result:
> 
> ```
> F0607 09:48:58.882109   11512 root.go:52] plugin "virt" not found
> ```
> 
> Does that relate to the plugin being installed only locally and not yet present in the krew-index or is this a bug?

It seems to be present though:

    $ kubectl krew list
    PLUGIN VERSION
    krew   dc2f2e1ec8a0acb6f3e23580d4a8b38c44823e948c40342e13ff6e8e12edb15a
    virt   bbf3b4f15b64ffe4c9196b53d045a92bc8113d24cfde471dded3c8c505aef663

Reply:
Yes it's because it's not in the index. We have a PR that saves the installed plugin receipt, it would eventually solve this problem.